api/tests/devcomp/unit/application/trainings/index_test.js

```diff
@@ -1090,46 +1090,46 @@ describe('Unit | Devcomp | Application | Trainings | Router | training-router',
     });
 
     describe('Security Prehandlers', function () {
-      // eslint-disable-next-line mocha/no-setup-in-describe
-      [
-        {
-          role: 'SUPER_ADMIN',
-          securityPreHandlersResponses: {
-            checkAdminMemberHasRoleSuperAdmin: (request, h) => h.response(true),
-            checkAdminMemberHasRoleMetier: (request, h) => h.response({ errors: new Error('forbidden') }).code(403),
-          },
-        },
-        {
-          role: 'METIER',
-          securityPreHandlersResponses: {
-            checkAdminMemberHasRoleSuperAdmin: (request, h) => h.response({ errors: new Error('forbidden') }).code(403),
-            checkAdminMemberHasRoleMetier: (request, h) => h.response(true),
-          },
-        },
-      ].forEach(({ role, securityPreHandlersResponses }) => {
-        it(`should verify user identity and return success update when user role is "${role}"`, async function () {
-          // given
-          sinon.stub(trainingController, 'createOrUpdateTrigger').returns('ok');
-          sinon
-            .stub(securityPreHandlers, 'checkAdminMemberHasRoleSuperAdmin')
-            .callsFake(securityPreHandlersResponses.checkAdminMemberHasRoleSuperAdmin);
-          sinon
-            .stub(securityPreHandlers, 'checkAdminMemberHasRoleMetier')
-            .callsFake(securityPreHandlersResponses.checkAdminMemberHasRoleMetier);
-          const httpTestServer = new HttpTestServer();
-          await httpTestServer.register(moduleUnderTest);
+      it(`should verify user identity and return success update when user role is "SUPER_ADMIN"`, async function () {
```

Comment:
beaucoup de choses ont été modifiées sur ce fichier dans cette PR, je vois bien le test qui doit retourner une 400 `should return bad request when tubes array is empty`, mais je ne comprends pas bien à quoi est dû le reste des modifications

Reply:
Check la section Remarques et les commits séparement ;)

Reply:
oh ok très clair merci :) approuvé